resources/js/components/Admin/Box.vue

```diff
@@ -101,7 +93,6 @@
     .top-node {
```

Comment:
Should refactor css for 4 nodes

Reply:
Should I reduce them to a single class, as most of the properties are duplicated? Something like this:

.node {
        position: absolute;
        height: 10px;
        width: 10px;
        background-color: #90ee90;
        cursor: grab;
    }

Reply:
Yes :D.

Reply:
done!